src/main/java/Tasks.java

```diff
@@ -28,7 +37,7 @@ public String toString() {
         }
         for(int i=0; i<this.length; i++)
         {
-            full_list += "\t" + (i+1) + "[" + (mark_as_done[i] ? "X" : " ") + "]" + list[i] + "\n";
+            full_list += "\t" + (i+1) +  ".[" + category[i] + "]" + "[" + (mark_as_done[i] ? "X" : " ") + "]" + list[i] + "\n";
```

Comment:
This line might be abit long. Can consider splitting it up into different parts and having one line to combine them all